treasury/RevenueSources.md

```diff
@@ -17,9 +17,10 @@ ChaosDAO's primary revenue sources are from running collators on various Dotsama
 
 * HydraDX collator - 7PAE9wNX6GohpdbNHtu7XYEp3oq5GiXbNAGBm6mXin1AYn3y
 
-Currently the ChaosDAO accounts hold about $15k.
 
-For more details, see the ChaosDAO Treasury Accounts spreadsheet at:
+Currently the ChaosDAO accounts hold about $15k
```

Comment:
Do we really want to publicize our treasury? Did we vote on this as DAOists and I just missed it?

If we did and that was the consensus decision, I'm fine with it...just my knee jerk reaction was that this shouldn't be shared

Reply:
no vote ever held. This piece evolved quite a few times as I wrote it. At first, had link to tracking spreadsheet, but I decided against that. Sometimes I'm writing this text to educate internal Regulars and Frens, but you're right that it's public. So I removed that line.